### ISSUE TITLE
No overrepresented genes (but I see significant p-values)

### ISSUE THREAD

**Issue by fcamus** [GuangchuangYu/ReactomePA#14]
Hello! 

First I need to say that I am really enjoying your packages for my RNASeq data! :) Thank you

I am trying out ReactomePA to look at my differentially expressed genes. I have previously used your other packages and was successful with this same dataset. I run the script:

`x <- enrichPathway(gene=defM.EB$table$ENTREZID,organism = "fly",qvalueCutoff=0.05, readable=T)`

when I just look at x I get no significance
`x`

"#...0 enriched terms found"

BUT  also tried 
`head(x)`

and it gave me an output with significant p and q values. 
ID   Description GeneRatio  BgRatio     pvalue     **p.adjust**
R-DME-72706  R-DME-72706   GTP hydrolysis and joining of the 60S ribosomal subunit    35/399  88/4280 8.672619e-15 **4.995429e-12**

(sorry, I dont know how to add R outputs nicely here)

I played around with changing the cutoff for the P/Q values, and I kept getting nothing being overrepresented. 

I just updated my R version to the latest (3.5.0)... would that be an issue? 

thank you!

**Comment by fcamus** [comment]
Hello! 

First I need to say that I am really enjoying your packages for my RNASeq data! :) Thank you

I am trying out ReactomePA to look at my differentially expressed genes. I have previously used your other packages and was successful with this same dataset. I run the script:

`x <- enrichPathway(gene=defM.EB$table$ENTREZID,organism = "fly",qvalueCutoff=0.05, readable=T)`

when I just look at x I get no significance
`x`

"#...0 enriched terms found"

BUT  also tried 
`head(x)`

and it gave me an output with significant p and q values. 
ID   Description GeneRatio  BgRatio     pvalue     **p.adjust**
R-DME-72706  R-DME-72706   GTP hydrolysis and joining of the 60S ribosomal subunit    35/399  88/4280 8.672619e-15 **4.995429e-12**

(sorry, I dont know how to add R outputs nicely here)

I played around with changing the cutoff for the P/Q values, and I kept getting nothing being overrepresented. 

I just updated my R version to the latest (3.5.0)... would that be an issue? 

thank you!
